style.js

```diff
@@ -1,20 +1,138 @@
+  },
+  {
+    project_id: 'project4',
+    name: 'Keeping track of hundreds of components',
+    description: ' Lorem Ipsum is simply dummy text of the printing  and typesetting industry. Lorem Ipsum has been the      industrys standard dummy text ever since the 1500s when an unknown printer took a galley of type and      scrambled it 1960s with the releaLorem Ipsum is simply      dummy text of the printing and typesetting industry.      Lorem Ipsum has been the industrys standard dummy text   ever since the 1500s, when an unknown      printer took a galley of type and scrambled it 1960s with the relea',
+    technologies: ['codekit', 'GitHub', 'Javascript', 'Bootstrap', 'terminal', 'Codepen'],
+    image: 'images/modal.png',
+    link: ['https://alfred-kinyua.github.io/Mobile-portfolio./', 'https://alfred-kinyua.github.io/Mobile-portfolio./'],
+  },
+  {
+    project_id: 'project5',
+    name: 'Keeping track of hundreds of components',
+    description: ' Lorem Ipsum is simply dummy text of the printing  and typesetting industry. Lorem Ipsum has been the      industrys standard dummy text ever since the 1500s when an unknown printer took a galley of type and      scrambled it 1960s with the releaLorem Ipsum is simply      dummy text of the printing and typesetting industry.      Lorem Ipsum has been the industrys standard dummy text   ever since the 1500s, when an unknown      printer took a galley of type and scrambled it 1960s with the relea',
+    technologies: ['codekit', 'GitHub', 'Javascript', 'Bootstrap', 'terminal', 'Codepen'],
+    image: 'images/modal.png',
+    link: ['https://alfred-kinyua.github.io/Mobile-portfolio./', 'https://alfred-kinyua.github.io/Mobile-portfolio./'],
+  },
+  {
+    project_id: 'project6',
+    name: 'Keeping track of hundreds of components',
+    description: ' Lorem Ipsum is simply dummy text of the printing  and typesetting industry. Lorem Ipsum has been the      industrys standard dummy text ever since the 1500s when an unknown printer took a galley of type and      scrambled it 1960s with the releaLorem Ipsum is simply      dummy text of the printing and typesetting industry.      Lorem Ipsum has been the industrys standard dummy text   ever since the 1500s, when an unknown      printer took a galley of type and scrambled it 1960s with the relea',
+    technologies: ['codekit', 'GitHub', 'Javascript', 'Bootstrap', 'terminal', 'Codepen'],
+    image: 'images/modal.png',
+    link: ['https://alfred-kinyua.github.io/Mobile-portfolio./', 'https://alfred-kinyua.github.io/Mobile-portfolio./'],
+  },
+];
+
+const openModalButtons = document.querySelectorAll('[data-modal-target]');
+const closeModalButtons = document.querySelectorAll('[data-close-button]');
+const overlay = document.getElementById('overlay');
+
+function openModal(modal) {
+  if (modal == null) return;
+
+  const modals = document.querySelectorAll('.modal');
+
+  for (let i = 0; i < projects.length; i += 1) {
```

Comment:
- [ ] Nice Job on getting the Desktop design, however, I kindly suggest that you follow the template design for the mobile design also as your modals buttons should be at the bottom and there should be a title directly underneath the image.

| Your Design  | Template Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/52934128/174074554-d4053675-f6df-466d-ad83-68f0940b5315.png)  | ![Screenshot from 2022-06-16 13-54-16](https://user-images.githubusercontent.com/52934128/174074665-d9d4ceb1-ec51-42ae-9ffd-da72d2e46d92.png)  |